src/config/liquidity.js

```diff
@@ -6,29 +6,56 @@ export default [
     logoPath: 'liquidity/banxa.svg',
     name: 'Banxa',
     description: 'Buy crypto with credit card directly to StarkNet',
-    url: {
-      [ChainType.L1.GOERLI]: '',
-      [ChainType.L1.MAIN]: 'https://starkware.banxa.com/'
+    link: {
+      [ChainType.L1.GOERLI]: {
+        url: '',
+        qsParams: {}
+      },
```

Comment:
Let's avoid defining unsupported chains... and handle the nullable value in the `Liquidty.js` when setting the url.

Reply:
Do you mean that `[ChainType.L1.GOERLI]: null`?